code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -76,6 +76,17 @@
 	turf_type = /turf/open/floor/grass
 	resistance_flags = FLAMMABLE
 
+/obj/item/stack/tile/grass/attackby(obj/item/W, mob/user, params)
+	if((W.tool_behaviour == TOOL_SHOVEL) && params)
+		to_chat(user, "<span class='notice'>You start digging up [src]</span>")
+		playsound(src, 'sound/effects/shovel_dig.ogg', 50, 1)
+		if(do_after(user, 2 * get_amount(), target = src))
+			new /obj/item/stack/ore/glass(get_turf(src), 2 * get_amount())
+			user.visible_message("[user] digs up [src].", "<span class='notice'>You uproot [src].</span>")
```

Comment:
```suggestion
			user.visible_message("<span class='notice'>[user] digs up [src].</span>", "<span class='notice'>You uproot [src].</span>")
```

Reply:
Again, this is a direct copy of grass turf behavior.